tcms/templates/menu_items.html

```diff
@@ -22,7 +22,7 @@
             {% endif %}
         </a>
         <ul class="dropdown-menu">
-            {% include "menu_items.html" with menu_items=item.1 dropdown_class="-submenu" %}
+            {% include "include/menu_items.html" with menu_items=item.1 dropdown_class="-submenu" %}
```

Comment:
This PR is missing the new file `tcms/templates/include/menu_items.html` while the old file `tcms/templates/menu_items.html` is still present. See `git mv` command.